src/keywords/optionalint.cpp

```diff
@@ -48,28 +49,42 @@ int OptionalIntegerKeyword::valueDelta() const { return valueDelta_; }
 // Return text to display in widget  when value is null
 std::string OptionalIntegerKeyword::textWhenNull() const { return textWhenNull_; }
 
+// Return keyword name
+std::string_view KeywordBase::name() const { return name_; }
+
 /*
  * Arguments
  */
 
 // Deserialise from supplied LineParser, starting at given argument offset
 bool OptionalIntegerKeyword::deserialise(LineParser &parser, int startArg, const CoreData &coreData)
 {
-    if (parser.hasArg(startArg))
+    try
     {
-        auto x = parser.argd(startArg);
+        if (!parser.hasArg(startArg))
+        {
+            return false;
+        }
```

Comment:
Stylistically, I prefer to not use curlies around simple statements like this when there's no need to enforce scope for what happens inside the braces.
```suggestion
            return false;
```